Tests/MessagingPushFCM/APITest.swift

```diff
@@ -30,10 +30,9 @@ class MessagingPushFCMAPITest: UnitTest {
 
         MessagingPush.shared.application(
             "",
-            didFailToRegisterForRemoteNotificationsWithError: CustomerIOError
-                .notInitialized
+            didFailToRegisterForRemoteNotificationsWithError: MessagingPushError.registrationFailed
```

Comment:
Replaced with generic error as `CustomerIOError` is no longer accessible by `MessagingPush`